src/Cache/Engine/RedisEngine.php

```diff
@@ -218,7 +218,12 @@ public function clear($check)
         }
         $keys = $this->_Redis->getKeys($this->_config['prefix'] . '*');
 
-        return !in_array(false, $this->deleteMany($keys));
+        $result = [];
+        foreach ($keys as $key) {
+            $result[] = $this->_Redis->delete($key) > 0;
```

Comment:
It looks like `delete()` takes an [array of keys](https://github.com/phpredis/phpredis#del-delete). Why are you calling delete N times?